Template/Pages/Long-term Staff.html

```diff
@@ -0,0 +1,10 @@
+<!--
+ Title: [* Προσωπικό */* Long-term Staff *]
+ -->
+[*
+<p>...</p>
+*/*
+<p> We are looking for people who: share the same values, will commit to Greek language learning, are flexible and adaptable, are self motivated, take initiative, persevere, can handle the different challenges that come with a pioneering situation, don’t compromise easily. </p>
```

Comment:
[`<ul>`](https://www.w3schools.com/tags/tag_ul.asp) defines a list. It is also easier to read. The style is adjustable.

```suggestion
<p> We are looking for people who:
<ul>
    <li>share the same values,</li>
    <li>will commit to Greek language learning,</li>
    <li>are flexible and adaptable,</li>
    <li>are self motivated,</li>
    <li>take initiative,</li>
    <li>persevere,</li>
    <li>can handle the different challenges that come with a pioneering situation, and</li>
    <li>don’t compromise easily.</li>
</ul>
</p>
```